server/src/test/java/io/crate/integrationtests/SnapshotRestoreIntegrationTest.java

```diff
@@ -811,6 +811,28 @@ public void test_restore_empty_partitioned_table_does_not_restore_other_tables()
             "empty_parted2");
     }
 
+    @Test
+    public void test_restore_table_does_not_restore_any_partitioned_table() {
```

Comment:
```suggestion
    @Test
    public void test_restore_table_does_not_restore_any_empty_partitioned_table() {
```

Reply:
afaik non-empty partitioned tables are not affected by this issue?

If so, let's reflect it in the test name

Reply:
Unfortunately not, it will always restore the template. The partitioned table is restored as empty as only the template is restored, I will re-phrase the changes entry.

Reply:
I've rephrased the changes and also adjusted the test case to cover both scenarios, empty and non-empty partitioned tables. Please check,

Reply:
Sorry for the late reply, got distracted a bit.

Have a minor comment on change entry, otherwise LGTM, thanks!

>Fixed a regression introduced with :ref:`version_5.6.0` that caused any
  partitioned table contained inside the snapshot to be restored, if not exists,
  by the :ref:`sql-restore-snapshot` statement 
  
`  if not exists` sounds confusing, can we drop it given that below we clarify that only definition was restored

Reply:
`if not exists` is even a SQL clause. That the definition only was restored is not related, it will be only restored if not exists.